### DISCUSSION TITLE
Listing issues and observation for Obsrv Deployment of release-7.0.0 in staging

### DISCUSSION THREAD

**Discussion by sanketnagdive** [Sunbird-Obsrv/Community#146] [Documentation]
This thread is created to list out all the issues faced while deploying the release-7.0.0 for obsrv . @sowmya-dixit  @ashokreddy1208
@gandham-santhosh @vinukumar-vs   @mohitga @rhwarrier @anandp504 @rajeevsathish
1. variables missing for datapipeline -> flinkpipelinejob job are
druid_storage_type, dp_ssl: false, dp_ssl_mode, dp_sslfactory, dp_db_name. It would be easier if these are there in release notes.
2. Analytics Core Build job is failing with the tag  so it was not working then tried creating fork and branch from the same tag listed in releasse note then it worked
observations: release notes does not pointing to the jobs so its confusing to know about the exact job(same jobs might be in different locations). Please let me know if you have any inputs

**Comment by sowmya-dixit** [comment]
@sanketnagdive 
1. We will get the release notes updated for any variable changes going forward.
2. Analytics Core build job failure looks like some configuration issue with Jenkins job. Please check if the build job is configured to use release tags or branches directly for building and get it fixed. Its not the issue with repository or the release tag, as same tag is being used in other environments.
